### ISSUE TITLE
Ex 8.2.3 unclear + how to prove divergence

### ISSUE THREAD

**Issue by ArthurAllshire** [pim-book/exercises#4]
I have submitted an erratum for this but for ex 8.2.3 I was wondering where it is intended to prove that the series diverges? Is it for a->infinity? (going by the notation of the definition in ex. 8.1.1)

If so I have been noodling around on this one for a while and I can't figure out how to prove it? I've tried via directly solving for k given M, but you just end up with a big mess of an equation that is hard to solve. Trying to show that the derivative goes to infinity as k goes to infinity (and thus that a_n grows without bound) also results in a big mess.

I know intuitively that the exponential in the numerator will eventually 'grow faster' than the polynomial in the denominator, but exactly how to prove this eludes me.

**Comment by j2kun** [comment]
This is a problem where estimation is the only reasonable way to go about it.

In particular, all we need to do is find a subsequence of a_n that diverges, which means we get the powerful flexibility to choose indexes that produce particularly simple sequence values.

For starters, you can choose n = m^2, and the subsequence is a_m = 2^m / m^20, which is simpler than before. Can you think of an additional (somewhat cleverer) choice that makes it easy to compare the top and bottom for which is bigger?

**Comment by j2kun** [comment]
I think it is also possible to use Taylor series to attack this problem, but (I think) it would require a Taylor expansion "at infinity" and I haven't _really_ explored this option. 

As a professor of mine used to say, that's like using the Millennium Falcon to shoot a sparrow.

**Comment by ArthurAllshire** [comment]
Question about this approach: is it actually sufficient to show that a subsequence diverges? For example if you had ![](https://latex.codecogs.com/gif.latex?n&space;=&space;m^2), and you proved that it diverged for ![](https://latex.codecogs.com/gif.latex?m&space;\in&space;\mathbb{N}) surely you cant then say that a_n diverges - because you have to prove that for *all* n > k that the sequence values are larger than a specific value?

Also, I've tried fiddling with this approach. The best I have been able to do is show that by choosing ![](https://latex.codecogs.com/gif.latex?n&space;=&space;(\log_{2}m)^2) you can reduce the problem to ![](https://latex.codecogs.com/gif.latex?a_m&space;=&space;\frac{m}{(log_{2}m)^{20}}), but still I run into the problem of actually showing that the bottom grows slower than the top.

**Comment by j2kun** [comment]
You're right that in general you can't use a subsequence to prove divergence, but in my opinion the numerator and denominator are simple and well-behaved enough that one could show if a subsequence diverges then the whole sequence diverges.

Perhaps a more direct approach would be to use the fact that log(x) is an increasing function, so that a_n diverges if and only if b_n = log(a_n) diverges. In this case we have reduced the problem to showing that the following diverges.

sqrt(n) - 10 log_2(n)

I think it is relatively straightforward to prove this from the definition: let M > 0 be as large as you need it to be to find an N(M) such that sqrt(N(M)) - 10 log_2(N(M)) > M
